src/SprykerEco/Shared/ComputopShipment/ComputopShipmentConstants.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Shared\ComputopShipment;
+
+/**
+ * Declares global environment configuration keys. Do not use it for other class constants.
+ */
+interface ComputopShipmentConstants
+{
+    /**
+     * Specification:
+     *  - Shipment method key for cart default shipment.
```

Comment:
```suggestion
     * - Shipment method key for cart default shipment.
```

please use single space